scripts/db-migration-test.sh

```diff
@@ -0,0 +1,27 @@
+TMPDIR=`mktemp -d 2>/dev/null || mktemp -d -t 'tmpConfigDir'`
+
+cp config/config.json $TMPDIR
+
+echo "Creating databases"
+docker run -v $(pwd)/tests:/sql -it --network host --rm mysql:5.7 sh -c 'exec mysql -h 127.0.0.1 -P 3306 -uroot -pmostest -e "CREATE DATABASE migrated; CREATE DATABASE latest; GRANT ALL PRIVILEGES ON migrated.* TO mmuser; GRANT ALL PRIVILEGES ON latest.* TO mmuser"'
```

Comment:
Could we leverage docker-compose instead to make tear down easier if something fails in the `Makefile` steps?

Reply:
Right now databases are created in mysql docker images already started by `start-docker` makefile command.
By tear down, do you mean deleting of those databases?
Does docker compose support running custom scripts before exiting?

Reply:
[Here](https://github.com/mattermost/mattermost-server/pull/10419/files#diff-63762a09c79f2ff5ac43373322d4ef79R27) is the tear down code which drops databases from locally running mysql instance

Reply:
If we spin up a new container for mysql instead of using already running instance then I guess tear down would be just removing that container, which I guess docker compose supports.
Is this what you meant?

Reply:
Ah, right -- I glossed over the fact that this uses the existing database. This is probably good enough to get us started: may have to tweak things once we integrate with the CI.

Reply:
@lieut-data Instead of running command inside a new container and then removing it, should I run commands inside already running container using `docker exec`? We are anyways using database running inside those containers.
That would look cleaner and I don't know if that would make integration to CI little easier?

The line linked here would change to:
`docker exec mattermost-mysql mysql -h 127.0.0.1 -P 3306 -uroot -pmostest -e "CREATE DATABASE migrated; CREATE DATABASE latest; GRANT ALL PRIVILEGES ON migrated.* TO mmuser; GRANT ALL PRIVILEGES ON latest.* TO mmuser"`

Reply:
👍 to running inside the existing container :)